CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.33.0]
+### Added
+- `sideEffects: false` in `package.json` to tell bundlers this package has no side effect so that it can be tree-shaken
+
```

Comment:
should we also listed the changes on `tsconfig` `compilerOptions` `target`?

Reply:
updated